secrets/keys

```diff
@@ -0,0 +1 @@
+HETZNER_DNS_API_TOKEN=
```

Comment:
This defiantly shouldn't be commited, if it's `.gitignore`d. If you want an give an example of how to do this, it should be named something like `secrets/keys.example`.

Reply:
Sure, that's a misunderstanding on my part. I saw that, when modified, `git add secrets` and `git add secrets/keys` would fail and assumed the same applied to `git add .`

Reply:
Git moment